src/app/components/shared/model-link/model-link.component.ts

```diff
@@ -0,0 +1,65 @@
+import {
+  ChangeDetectionStrategy,
+  Component,
+  Input,
+  OnChanges,
+} from "@angular/core";
+import { isInstantiated } from "@helpers/isInstantiated/isInstantiated";
+import { AbstractModel } from "@models/AbstractModel";
+import { User } from "@models/User";
+
+// TODO Pass model to content through context
+@Component({
+  selector: "baw-model-link",
+  template: `
+    <!-- Loading text -->
+    <ng-container *ngIf="model | isUnresolved; else resolved">
+      <ng-content select="#unresolved"></ng-content>
+    </ng-container>
+
+    <ng-template #resolved>
+      <!-- Normal model with view url -->
+      <a *ngIf="!isGhostUser && hasViewUrl" [bawUrl]="model.viewUrl">
```

Comment:
inappropriate variable name `isGhostUser` in the case that this now a general component

Reply:
I'm unsure what a more appropriate name is? `isUnknownOrDeletedUserModel`? `isGhostUser` identifies that this is a user model of type ghost. From your other comment below, it seems you just don't want this state being tracked